resources/testharness.js

```diff
@@ -1476,6 +1476,9 @@ policies and contribution forms [3].
         if (tests.file_is_test && tests.tests.length) {
             throw new Error("Tried to create a test with file_is_test");
         }
+        if (properties.timeout) {
```

Comment:
I had this in my patch to look for tests that need updating, but I think we should just ignore `properties.timeout` when this change lands. In other words, I suggest removing this block of code.

Some time after `properties.assert` has also been removed, I could do another run to see if any tests pass properties at all, and then finally we'll just ignore extra arguments if they are given.

Reply:
Removed.